src/domain/app/i18n/en.json

```diff
@@ -286,6 +286,7 @@
       "imageAltTextHelp": "Description of the contents of the image. Add license information also.",
       "labelDescription": "Description",
       "labelMandatoryAdditionalInformation": "Entering additional information is mandatory for the registrant",
+      "guidanceTextMandatoryAdditionalInformation": "Tell above if you need more information about group or other things in activities and make answer as mandatory...",
```

Comment:
rather than using guidanceText, helperText would be more in line with other translations and the prop name

Reply:
I selected guidance text only because it's not an actual helperText. Checkboxes seems to not have them. Also that text is related to 2 fields. But I don't mind the name, I can change it if needed.

Reply:
OK, no need to change :)